src/blockdata/block.rs

```diff
@@ -242,6 +243,42 @@ impl Block {
         let txs_weight: usize = self.txdata.iter().map(Transaction::get_weight).sum();
         base_weight + txs_weight
     }
+
+    /// Get the coinbase transaction, if one is present.
+    pub fn coinbase(&self) -> Option<&Transaction> {
+        self.txdata.iter().next()
```

Comment:
There is a function `Vec::first` which does this

Reply:
done